src/plugins/vis_default_editor/public/components/controls/number_interval.tsx

```diff
@@ -78,23 +117,32 @@ function NumberIntervalParamEditor({
       isInvalid={showValidation && !isValid}
       helpText={get(editorConfig, 'interval.help')}
     >
-      <EuiFieldNumber
-        value={value === undefined ? '' : value}
-        min={min}
-        step={base}
-        data-test-subj={`visEditorInterval${agg.id}`}
-        isInvalid={showValidation && !isValid}
-        onChange={onChange}
-        onBlur={setTouched}
-        fullWidth={true}
-        compressed
-        placeholder={i18n.translate(
-          'visDefaultEditor.controls.numberInterval.selectIntervalPlaceholder',
-          {
-            defaultMessage: 'Enter an interval',
-          }
-        )}
-      />
+      <EuiFlexGroup gutterSize="s" responsive={false} direction={'column'}>
+        <EuiFlexItem>
+          <EuiSwitch
+            label={useAutoIntervalLabel}
+            onChange={onAutoSwitchChange}
+            checked={autoChecked}
+            compressed
+            data-test-subj={`visEditorIntervalSwitch${agg.id}`}
+          />
+        </EuiFlexItem>
+        <EuiFlexItem>
+          <EuiFieldNumber
+            value={value}
```

Comment:
The `auto` value produce warnings to a browser console:
![image](https://user-images.githubusercontent.com/31325372/91726985-b95b7300-eba9-11ea-9420-4da9ad9d35b9.png)

Reply:
Thank you!